src/index.css

```
@@ -336,10 +338,15 @@
 
-  .glass-input:focus {
+  .glass-input:focus,
+  .glass-input.isOpen {
     outline: none;
     @apply border-none;
-    box-shadow: var(--input-shadow), var(--glass-rim-subtle), 0 0 0 2px rgb(var(--accent-500) / 0.3);
+    box-shadow:
+      var(--input-shadow),
+      var(--glass-rim-subtle),
+      0 0 0 2px rgb(var(--accent-500) / 0.25),
+      0 0 12px rgb(var(--accent-500) / 0.15);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)